src/core_modules/capture-core/components/Pages/MainPage/TeiWorkingLists/ReduxProvider/TeiWorkingListsReduxProvider.container.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { TeiWorkingListsSetup } from '../Setup';
+import { useWorkingListsCommonStateManagement } from '../../WorkingListsCommon';
+import { useTrackerProgram } from '../../../../../hooks/useTrackerProgram';
+import { TEI_WORKING_LISTS_TYPE } from '../constants';
+import type { Props } from './teiWorkingListsReduxProvider.types';
+
+export const TeiWorkingListsReduxProvider = ({ storeId }: Props) => {
+    const programId = useSelector(({ currentSelections }) => currentSelections.programId);
```

Comment:
Thats more of a thought than a suggestion to change the code. 

I am thinking if it makes any sense to create its own hook from this one since if I find myself writing the same selector over and over. What you think?

Reply:
Hmm. Probably more work to import this one from somewhere else though, and very minor readability improvement. But I won't stop you doing it ;)

Reply:
Yeah dont know if there is value here since this is indeed too small to do it anyway. But perhaps if in the future if I use it 100 times I might do it in the end.